<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorTestForm</class>
 <widget class="QWidget" name="ColorTestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label0">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ishihara Color Blind Test</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>There are 15 images. For each image, you may recognize a number. Please complete the questions below.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labImageNumber">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Test images</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labPixture">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rbnNothing">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>No number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbnNumber">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditNumber"/>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnThird">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbtnLastQues">
        <property name="text">
         <string>Last</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbtnNextQues">
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
